<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>632</width>
          <height>953</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>General</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="7" column="1">
             <widget class="QSpinBox" name="spinBoxDisplayBackground">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QLabel" name="labelDisplayBackgroundExplain">
              <property name="text">
               <string>(of white)</string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLabel" name="labelSuffixPercent2">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="labelSelector">
              <property name="text">
               <string>Selection tool:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxFitWindowWhenZoomed">
              <property name="text">
               <string>Fit window to images when zoomed</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="comboBoxFileOrder">
              <item>
               <property name="text">
                <string>Name</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Name (reversed)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Time</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Time (reversed)</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="14" column="3">
             <widget class="QLabel" name="labelDisplayQualityExplain">
              <property name="text">
               <string>(reduces downscaling aliasing effects)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelFileOrder">
              <property name="text">
               <string>File order:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="14" column="1">
             <widget class="QComboBox" name="comboBoxDisplayQuality">
              <item>
               <property name="text">
                <string>Fast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quality</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="labelSuffixPercent1">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxReverseMouseWheel">
              <property name="text">
               <string>Reverse mouse wheel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelDisplayMode">
              <property name="text">
               <string>Display mode:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QSpinBox" name="spinBoxZoomDelta">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="10" column="3">
             <widget class="QLabel" name="labelClearClipboardExplain">
              <property name="text">
               <string>(if disabled, the exit could take a long time)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxLoopDir">
              <property name="text">
               <string>Loop directory</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="labelDisplayBackground">
              <property name="text">
               <string>Display background:</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QComboBox" name="comboBoxSelector">
              <item>
               <property name="text">
                <string>XOR frame</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Highlight (fast)</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="11" column="3">
             <widget class="QLabel" name="labelSelectorExplain">
              <property name="text">
               <string>(not applied on the current image)</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxClearClipboard">
              <property name="text">
               <string>Clear Clipboard on exit</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="labelZoomDelta">
              <property name="text">
               <string>Zoom delta:</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxEnableToolbarShrinking">
              <property name="text">
               <string>Enable Toolbar shrinking</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="comboBoxDisplayMode">
              <item>
               <property name="text">
                <string>Fit window to images</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit all images to window</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit large images to window</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Do not fit</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit images to desktop</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit large images to desktop</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxStartMaximized">
              <property name="text">
               <string>Start maximized</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="labelDisplayQuality">
              <property name="text">
               <string>Display quality:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxLocale">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>System</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelLocale">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Locale:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="4">
             <widget class="QCheckBox" name="checkBoxStartFullscreen">
              <property name="text">
               <string>Start fullscreen (if image is loaded on startup)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxFs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Fullscreen</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="6" column="0">
             <widget class="QLabel" name="labelFsIndicatorFontSize">
              <property name="text">
               <string>Indicator font size:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxFsFileIndexIndicator">
              <property name="text">
               <string>Enable file index indicator</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="comboBoxFsIndicatorColor">
              <item>
               <property name="text">
                <string>Dark gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Light gray</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark Blue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelFsDisplayMode">
              <property name="text">
               <string>Display mode:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxFsDisplayMode">
              <item>
               <property name="text">
                <string>Fit all images to display</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fit large images to display</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxFsFileNameIndicator">
              <property name="text">
               <string>Enable file name indicator</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="labelSuffixPixel1">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxFsHideCursor">
              <property name="text">
               <string>Hide cursor</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="spinBoxFsIndicatorFontSize">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>8</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelFsIndicatorColor">
              <property name="text">
               <string>Indicator color:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QFontComboBox" name="fontComboBoxFsIndicatorFontFamily"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelFsIndicatorFontFamily">
              <property name="text">
               <string>Indicator font:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxUs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Undo stack</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxUsMinimumSteps">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelUsMemoryLimit">
              <property name="text">
               <string>Memory limit:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelUsMinimumSteps">
              <property name="text">
               <string>Minimum number of steps:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxUsMemoryLimit">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65536</number>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="labelSuffixMegaByte1">
              <property name="text">
               <string>MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxTh">
           <property name="title">
            <string>Thumbnails</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBoxThScrollSpeed">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxThCacheSize">
              <property name="minimum">
               <number>512</number>
              </property>
              <property name="maximum">
               <number>8192</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelThScrollSpeed">
              <property name="text">
               <string>Scroll speed:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelThCacheSize">
              <property name="text">
               <string>Cache size:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelThMultithreading">
              <property name="text">
               <string>Multithreading:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxThMultithreading">
              <item>
               <property name="text">
                <string>Auto</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>64</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxThPreloading">
              <property name="text">
               <string>Preloading</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>comboBoxLocale</tabstop>
  <tabstop>checkBoxStartMaximized</tabstop>
  <tabstop>comboBoxFileOrder</tabstop>
  <tabstop>checkBoxLoopDir</tabstop>
  <tabstop>comboBoxDisplayMode</tabstop>
  <tabstop>checkBoxFitWindowWhenZoomed</tabstop>
  <tabstop>spinBoxDisplayBackground</tabstop>
  <tabstop>spinBoxZoomDelta</tabstop>
  <tabstop>checkBoxReverseMouseWheel</tabstop>
  <tabstop>checkBoxClearClipboard</tabstop>
  <tabstop>checkBoxFsHideCursor</tabstop>
  <tabstop>comboBoxFsDisplayMode</tabstop>
  <tabstop>checkBoxFsFileIndexIndicator</tabstop>
  <tabstop>checkBoxFsFileNameIndicator</tabstop>
  <tabstop>comboBoxFsIndicatorColor</tabstop>
  <tabstop>fontComboBoxFsIndicatorFontFamily</tabstop>
  <tabstop>spinBoxFsIndicatorFontSize</tabstop>
  <tabstop>spinBoxUsMinimumSteps</tabstop>
  <tabstop>spinBoxUsMemoryLimit</tabstop>
  <tabstop>spinBoxThCacheSize</tabstop>
  <tabstop>checkBoxThPreloading</tabstop>
  <tabstop>comboBoxThMultithreading</tabstop>
  <tabstop>spinBoxThScrollSpeed</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
